### DISCUSSION TITLE
Dart or() Foreign Table

### DISCUSSION THREAD

**Discussion by angus-clark** [supabase/supabase#7797] [Questions] [answered]
There seems to be no "Use or on foreign tables" for Dart as there is in javascript as outlined here in the docs https://supabase.com/docs/reference/javascript/or#use-or-on-foreign-tables

Will this be added?

**Comment by dshukertjr** [comment, marked as answer]
Hi @clarky2233,

Thanks for starting this discussion!
I have created an issue [here](https://github.com/supabase-community/supabase-flutter/issues/151) to track it. It shouldn't take too long until this is added to Dart libraries!
